adoc/quickstart3_chap_suma_keys_and_first_client.adoc

```diff
@@ -318,31 +321,38 @@ The feature is not currently supported on {rhel} or Salt clients.
 . Navigate to the menu:Software[Packages > Lock] tab on the managed system to see a list of all available packages.
 . Select the packages to lock, and click btn:[Request Lock].
 You can also choose to enter a date and time for the lock to activate.
-Note that even if you do not select a date and time, the lock might not activate immediately.
+Leave the date and time blank if you want the lock to activate as soon as possible.
+Note that the lock might not activate immediately.
 . To remove a package lock, select the packages to unlock and click btn:[Request Unlock].
-You can also choose to enter a date and time for the lock to deactivate.
-Note that even if you do not select a date and time, the lock might not deactivate immediately.
+Leave the date and time blank if you want the lock to deactivate as soon as possible.
+Note that the lock might not deactivate immediately.
 
 
 
 
 [[preparing.and.registering.clients.salt]]
 == Registering Salt Clients
 
-There are currently three methods for registering Salt minions.
-This section describes the first method and uses a bootstrap repository.
-The second method uses the bootstrap script, and is mostly similar to the procedure described in <<registering.clients.traditional>>{mdash}the difference is enabling [guimenu]``Bootstrap using Salt`` and the activation key option [guimenu]``Configuration File Deployment`` that applies highstate automatically.
-The third method uses the {webui}, and is described in <<ref.webui.systems.bootstrapping>>.
+There are three possible methods for registering Salt minions.
+You can use a bootstrap repository, a bootstrap script, or install using the {webui}.
 
-You can also use these methods to change existing traditional clients into Salt minions.
+This section describes using a bootstrap repository.
+Registering Salt clients with a bootstrap client is the same as registering traditional clients, which is described at xref:quickstart3_chap_suma_keys_and_first_client.adoc#registering.clients.traditional[Registering Traditional Clients]. When using this method, ensure you enable the [guimenu]``Bootstrap using Salt`` and activation key options in [guimenu]``Configuration File Deployment``, so that highstate is applied automatically.
+For information on using the {webui}, see xref:reference-webui-systems.adoc#ref.webui.systems.bootstrapping[Bootstrapping Salt].
 
+You can also use these methods to change existing traditional clients into Salt minions.
 
-The rest of this section assumes you have created a SUSE Manager tools repository.
-You can review creating a tools repository in <<create.tools.repository>>.
+[IMPORTANT]
+.GPG Keys and Uyuni Client Tools
+====
+The GPG key used by Uyuni Client Tools is not trusted by default.
+Either update your bootstrap repository to trust the key explicitly, or use the {webui} to manually trust the key from each client.
```

Comment:
Should we add a suggestion about how to do it? With configuration channels, I guess.

Reply:
I was OK with leaving it vague just because there are several ways to do it, and I'm fairly certain they're all documented elsewhere.